### ISSUE TITLE
[A11y] [Splitter] Incorrect JAWS announcements when tabbing through Splitter

### ISSUE THREAD

**Issue by tynan-dev** [telerik/kendo-angular#4032]
**Describe the bug**
JAWS screen reader incorrectly reads out "To move through items press the up or down arrow" when landing on the Splitter. On pressing Enter, JAWS should announce whether the pane has been expanded or collapsed, but it only says "Enter."

**To Reproduce**

1. Implement a basic Splitter ([StackBlitz](https://stackblitz.com/edit/angular-pngj8u?file=src%2Fapp%2Fapp.component.ts))
2. Start JAWS screen reader
3. Tab through and listen to JAWS readout

**Expected behavior**
JAWS should instruct the user to use the left and right arrow keys instead of up and down. On pressing Enter, JAWS should announce that the pane has been expanded/collapsed.

**Browser**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 114

**Comment by slavenai** [comment]
Hi @tynan-dev ,

Announcing which arrow keys should be used would indeed be a nice enhancement. We will discuss it internally and update our accessibility specification accordingly.

Announcing the pane `expanded` state,  however, would be inaccurate as [separators](https://www.w3.org/TR/wai-aria-1.2/#separator) don't support such state.
